tests/mpls/test_mpls.py

```diff
@@ -0,0 +1,330 @@
+import logging
+import os
+import pprint
+import ptf.mask as mask
+import ptf.packet as packet
+import ptf.testutils as testutils
+import pytest
+import time
+
+from abc import ABCMeta, abstractmethod
+
+logger=logging.getLogger(__name__)
+
+pytestmark=[
+    pytest.mark.topology('t1'),
```

Comment:
Can this test run against any regular SONiC image built from master branch? Please call out any missing feature dependency.

Reply:
If the platform supports MPLS then master branch image should be working on the platform. The script will run if the SAI for MPLS is implemented. Feature is not dependent on other features.